Handler/Plan.hs

```diff
@@ -0,0 +1,22 @@
+module Handler.Plan where
+
+import Import
+
+getPlansR :: Handler Html
+getPlansR = do
+    plans <- runDB $ selectList [] [Asc PlanSort]
```

Comment:
https://www.fpcomplete.com/hoogle?q=selectList&env=ghc-7.8-stable-14.09

It selects all plans matching no filters (first argument), and sorting by `PlanSort` ascending (second argument).